### PR TITLE
[webui] make use of 'sprite_tag' for build status icon

### PR DESCRIPTION
**Pull request by mschnitzer**
This fixes an issue that the icon was not correctly rendered in production mode.

### CODE REVIEW

File: src/api/app/helpers/webui/buildstatus_helper.rb [hunk 1]
```diff
@@ -22,7 +22,7 @@ module Webui::BuildstatusHelper
   # rubocop:enable Style/AlignHash
 
   def get_package_status_description(status)
-    PACKAGE_STATUS_LIST[status] || "status explanation not found"
+    PACKAGE_STATUS_LIST[status.to_s] || "status explanation not found"
```
Comment (hennevogel) [inline src/api/app/helpers/webui/buildstatus_helper.rb]: Why to_s?

**Comment by hennevogel** [comment]
This has nothing to do with the API btw...

File: src/api/app/helpers/webui/buildstatus_helper.rb (hunk 1 above)
Comment (mschnitzer) [inline src/api/app/helpers/webui/buildstatus_helper.rb]: Because status is not always a string.

**Comment by mschnitzer** [comment]
@hennevogel my bad. I change it to `[webui]`

**Comment by mschnitzer** [comment]
Superseded by #2011

**Pull request closed without merging**